bugz.js

```diff
@@ -83,6 +83,10 @@ class BugzillaBug extends Bug {
   get project() {
     return this._data.component;
   }
+
+  get mentors() {
+    return this._data.mentors;
```

Comment:
Add an implementation in the base class that just returns `[]`?
That should make things more clean.